tests/robot-tests/tests/admin/bau/prerelease.robot

```diff
@@ -369,7 +351,6 @@ Go back to prerelease content page again
 Go to prerelease table tool page
     [Tags]  HappyPath
     user clicks link  Table tool
```

Comment:
In other instances we're using `user waits until h2 is visible View a featured table or create your own` after clicking the Table tool link - so I think we should also do that here?